bindings/rust/integration/benches/s2nc.rs

```diff
@@ -0,0 +1,40 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use criterion::{criterion_group, criterion_main, Criterion};
+use std::{
+    dbg, env,
+    io::{self, Write},
+    process::Command,
+    time::Duration,
+};
+mod utils;
+
+pub fn s2nc(c: &mut Criterion) {
+    let mut group = c.benchmark_group("s2nc");
+    let s2nc_env: &str = &env::var("S2NC_ARGS").unwrap();
+    let s2nc_args: utils::Arguments = s2nc_env.into();
+    let test_name = format!("s2nc_{}", s2nc_args.get_endpoint().unwrap());
+    dbg!("Parsed test_name as: {:?}", &test_name);
+    group.bench_function(test_name, move |b| {
+        b.iter(|| {
+            let s2nc_env: &str = &env::var("S2NC_ARGS").unwrap();
+            let s2nc_args: utils::Arguments = s2nc_env.into();
+            dbg!("s2nc harness: {:?}", &s2nc_args);
+            let output = Command::new("/usr/local/bin/s2nc")
+                .args(s2nc_args.get_vec())
+                .output()
```

Comment:
Rather than calling output, I would call status since stdio is automatically inherited from the parent process

https://doc.rust-lang.org/std/process/struct.Command.html#method.status